tree.txt

```diff
@@ -29,37 +29,36 @@
 [INFO] |  +- org.jboss.shrinkwrap:shrinkwrap-api:jar:1.2.3:compile
 [INFO] |  \- org.jboss.shrinkwrap:shrinkwrap-spi:jar:1.2.3:compile
 [INFO] +- org.slf4j:slf4j-api:jar:1.7.21:compile
-[INFO] +- org.apache.jclouds.provider:aws-ec2:jar:2.1.1:compile
-[INFO] |  +- org.apache.jclouds.api:ec2:jar:2.1.1:compile
-[INFO] |  |  \- org.apache.jclouds.api:sts:jar:2.1.1:compile
-[INFO] |  \- com.google.errorprone:error_prone_annotations:jar:2.1.0:compile
-[INFO] +- org.apache.jclouds.api:docker:jar:2.1.1:compile
-[INFO] |  +- org.apache.jclouds:jclouds-core:jar:2.1.1:compile
-[INFO] |  |  +- javax.ws.rs:javax.ws.rs-api:jar:2.0.1:compile
-[INFO] |  |  +- com.google.inject.extensions:guice-assistedinject:jar:3.0:compile
-[INFO] |  |  +- com.google.inject:guice:jar:3.0:compile
+[INFO] +- org.apache.jclouds.provider:aws-ec2:jar:2.4.0:compile
```

Comment:
I think `tree.txt` got accidently into the commit. Could you remove it, please?

Reply:
fixed